sui/deploy-gateway.js

```diff
@@ -126,6 +159,14 @@ if (require.main === module) {
     program.addOption(new Option('--minimumRotationDelay <minimumRotationDelay>', 'minium delay for signer rotations (in ms)').default(0));
     program.addOption(new Option('--domainSeparator <domainSeparator>', 'domain separator').default(HashZero));
     program.addOption(new Option('--nonce <nonce>', 'nonce for the signer (defaults to HashZero)'));
+    program.addOption(new Option('--previousSigners <previousSigners>', 'number of previous signers to retain').default(0));
+    program.addOption(new Option('--upgradeCap <upgradeCap>', 'gateway UpgradeCap id'));
+    program.addOption(new Option('--upgrade', 'upgrade a deployed contract'));
+    program.addOption(new Option('--policy <policy>', 'new policy to upgrade'));
```

Comment:
not clear what policy arg looks like, add example in help text and use readable .choices() restriction?